dockerbot.py

```diff
@@ -26,13 +48,25 @@ def handle(msg):
     if command == '/sign':
         v_Kid = "sign"
         try:
-            subprocess.check_output(['python', '/etc/Health_Statements.py', '-u', v_UserId, '-p', v_UserKey, '-k', v_Kid])
+            if v_SIGN_PARENTS_EDUCATION_GOV_IL == 1:
+                bot.sendMessage(chat_id,"Starting Sign process at https://parents.education.gov.il")
+                subprocess.check_output(['python', '/etc/Health_Statements.py', '-u', v_UserId, '-p', v_UserKey, '-k', v_Kid])
+
+            if v_SIGN_WEBSITE_MASHOV == 1:
+                if v_MASHOV_NUMBER_OF_KIDS > 0:
+                    for Mashov_Kid_Number in range(1, v_MASHOV_NUMBER_OF_KIDS, 1):
+                        bot.sendMessage(chat_id,"Starting Sign process at https://web.mashov.info/students/login for Kid Number -" + str(Mashov_Kid_Number))
+                        Prep_Switch_MASHOV_USER_DICT_ID_KID = v_MASHOV_USER_DICT_ID_KID[Mashov_Kid_Number]
+                        Prep_Switch_MASHOV_USER_DICT_ID_PWD = v_MASHOV_USER_DICT_ID_PWD[Mashov_Kid_Number]
+                        Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID = v_MASHOV_USER_DICT_ID_SCHOOL_ID[Mashov_Kid_Number]
+                        subprocess.check_output(['python', '/etc/Mashov_Health_Statements.py', '-u', Prep_Switch_MASHOV_USER_DICT_ID_KID, '-p', Prep_Switch_MASHOV_USER_DICT_ID_PWD , '-s', Prep_Switch_MASHOV_USER_DICT_ID_SCHOOL_ID, '-kn', str(Mashov_Kid_Number)])
+
```

Comment:
@UdiMizrachi  please seprate the education and the mashov in to 2 seperate commands:
example:
/sign_edu
/sign_mashov
feel free to set diffrent names.

Reply:
Done,
Also added logic to the Mashov website to check if checkmarks already selected (At Mashov website you can remove/add checkmarks all day long, not as EDU website which is 1 run per day)